### PR TITLE
Add get processes in a group

### PR DESCRIPTION
**Pull request by mmb**
Co-authored-by: J.P. Poveda <etsuo@users.noreply.github.com>

### CODE REVIEW

**Comment by mmb** [comment]
This is a preliminary PR for adding support for the measurements API. Our goal is to use the measurements API to write a prometheus exporter for Atlas.

File: mongodbatlas/processes.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	Port           int     `json:"port"`
+	ShardName      string  `json:"shardName"`
+	ReplicaSetName string  `json:"replicaSetName"`
+	TypeName       string  `json:"typeName"`
+	Version        string  `json:"version"`
+}
+
+type processesResponse struct {
+	Links      []*Link   `json:"links,omitempty"`
+	Results    []Process `json:"results,omitempty"`
```
Comment (gssbzn) [inline mongodbatlas/processes.go]: [nit] We try to keep the results as pointers, this is also necessary for the `omitempty` in this case

File: mongodbatlas/processes.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+const processesPath = "groups/%s/processes"
+
+type ProcessesService interface {
```
Comment (gssbzn) [inline mongodbatlas/processes.go]: some small nits with all public methods/structs, we try to keep then documented and following and with links to the api docs (https://docs.atlas.mongodb.com/reference/api/processes-get-all/ in this case) you can follow some other implementations as a reference of how we format these docs

**Comment by mmb** [comment]
Thank you for review! I have added the comments, used pointers and signed the CLA.

Please let us know if there is anything else that can be improved.

**Comment by mmb** [comment]
Thanks all. Great news about the measurements API.

**Comment by themantissa** [comment]
@mmb can you tidy up based on the CI results so we can merge?  Thanks!

**Comment by themantissa** [comment]
@gssbzn thank you so much for that!   @mmb all good now!

**Pull request merged**